### DISCUSSION TITLE
[How to use] How to save image to album in iOS

### DISCUSSION THREAD

**Discussion by kanari3** [fluttercandies/flutter_photo_manager#1000] [Q&A] [answered]
### Platforms

iOS

### Description

About iOS
Is it possible to add image to created album in photo_manager 2.7.1 ?
If you don't support saving and adding to an album, please let me know how to add to a folder.

Could this be implemented in the future?
Are there alternative solutions?

### My code

I can create an album with the following code, but I don't know how to add images to the album I created.

```
PhotoManager.editor.darwin.createAlbum();
```

### Try do it

I referred to the following thread
https://github.com/fluttercandies/flutter_photo_manager/issues/865

**Comment by CaiJingLong** [comment, marked as answer]
In iOS, all Assets are stored in a huge container, and Album is more like adding a Tag to an Asset.
So, you need to use `saveImage*` or `saveVideo*` to save the Asset first, and use the `copyAssetToPath` method.
If album does not exist, you also need to use the `createAlbum` method before copying.

**Reply by kanari3** [reply]
Thanks, I was able to save the images in an album!
I have created a sample app and hope it will be useful to someone in the future!
https://github.com/kanari3/save_image_to_album_flutter
